presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/SymbolAliases.java

```diff
@@ -197,7 +219,6 @@ public Builder put(String alias, SymbolReference symbolReference)
             }
 
             checkState(!bindings.containsKey(alias), "Alias '%s' already bound to expression '%s'. Tried to rebind to '%s'", alias, bindings.get(alias), symbolReference);
-            checkState(!bindings.values().contains(symbolReference), "Expression '%s' is already bound in %s. Tried to rebind as '%s'.", symbolReference, bindings, alias);
```

Comment:
@ebd2 I got rid of this check entirely because there wasn't a good way to distinguish the manually created bindings from subsequent automatically generated ones. And it didn't seem like it was getting us all that match.